admin_app_2/src/components/ContentCard.tsx

```diff
@@ -0,0 +1,88 @@
+import ContentViewModal from "@/components/ContentModal";
+import { LANGUAGE_OPTIONS, appColors, appStyles, sizes } from "@/utils";
+import { Edit, Translate } from "@mui/icons-material";
+import { Button, Card, Typography } from "@mui/material";
+import Link from "next/link";
+import React from "react";
+import { Layout } from "./Layout";
+
+const ContentCard = ({
+  title,
+  text,
+  content_id,
+  last_modified,
+  onDelete,
+}: {
+  title: string;
+  text: string;
+  content_id: string;
+  last_modified: string;
+  onDelete: () => void;
+}) => {
+  const [open, setOpen] = React.useState<boolean>(false);
+  return (
+    <>
+      <Card
+        sx={[
+          {
+            m: sizes.smallGap,
+            p: sizes.baseGap,
+          },
+          appStyles.shadow,
+        ]}
+      >
+        <Typography variant="h6">{title}</Typography>
```

Comment:
We should stick to the wireframe design unless strictly necessary since that's where we all align on functionality - I agree that that might be helpful but I don't think we should make that decision during implementation

Reply:
I changed a few things (moving the ID down to bottom right of the cards, moving add button to top right of the cardsgrid) which I shouldn't have as well.